src/generators/web.cr

```diff
@@ -9,8 +9,11 @@ class LuckyCli::Generators::Web
   def initialize(
     project_name : String,
     @api_only : Bool,
-    @generate_auth : Bool
+    @generate_auth : Bool,
+    project_directory : String = "."
   )
+    Dir.cd(File.expand_path(project_directory))
```

Comment:
The `expand_path` allows us to pass things like `"."` or `~/Projects` and have it expand properly. We just cd in to the directory we want the project, then run everything as normal.